bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/LFV_TQMuTau/LFV_ST_TQMuTau/LFV_ST_TQMuTau_customizecards.dat

```diff
@@ -0,0 +1,2 @@
+set param_card mqt 172.5
```

Comment:
Is it really called mqt or a typo?

Reply:
https://github.com/cms-sw/genproductions/pull/2825/files#diff-753be9769c41ac94b94a9994e0690d427b6443dbbdd35e2068a37c047c47c2d9R939

This looks like what mass of top quark is named in this model

Reply:
I can update customizecard with using blocks in parameter cards.
```
set param_card mass 6 172.5
```

Reply:
No, no. I just wanted to double-check assuming you would know on top of your head and I don't need to verify myself. If that is the name it is the name and all fine.

Reply:
Okay, I understood. The update version also works, so it would be okay.